### ISSUE TITLE
Good work!

### ISSUE THREAD

**Issue by qu10wenhao** [Peter-Suchang/NumGuess#3]
Well, the developer's idea is very close to another 2 developers. the code is short and efficient and has no errors. The English notes explain the use of each function very clearly.
Here is my advice. It's better to use 2 arrays instead of 6 variables to save the random numbers and the guess numbers. And using loop to judge the values of A and B is better, so that this program can also be fit for the large Guessnum game.
in conclusion, this program successfully achieve the goal. Well done!
